artifact/v3/manifest_sig/manifest_sig.hpp

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#ifndef MENDER_ARTIFACT_V3_MANIFEST_SIG_PARSER_HPP
+#define MENDER_ARTIFACT_V3_MANIFEST_SIG_PARSER_HPP
+
+#include <string>
+
+#include <common/expected.hpp>
+#include <common/error.hpp>
+#include <common/io.hpp>
+
+namespace mender {
+namespace artifact {
+namespace v3 {
+namespace manifest_sig {
+
+using namespace std;
+
+namespace expected = mender::common::expected;
+namespace io = mender::common::io;
+namespace error = mender::common::error;
+
+class ManifestSignature {
+public:
+	ManifestSignature(const std::string &sig) :
+		value(sig) {
+	}
+	std::string value;
+};
```

Comment:
What is this class good for? It's basically the same as `using ManifestSignature = string`.

Reply:
@oleorhagen @vpodzime I'll convert into struct :+1: